pkg/operator/ceph/object/config.go

```diff
@@ -215,12 +216,39 @@ func (c *clusterConfig) generateMonConfigOptions(rgwConfig *rgwConfig) (map[stri
 	for flag, val := range c.store.Spec.Gateway.RgwConfig {
 		if currVal, ok := configOptions[flag]; ok {
 			// RGW might break with some user-specified config overrides; log clearly to help triage
-			logger.Infof("overriding object store %q RGW config option %q (current value %q) with user-specified rgwConfig %q",
+			logger.Infof("overriding object store %q RGW config option %q with user-specified rgwConfig",
+				fmt.Sprintf("%s/%s", c.store.Namespace, c.store.Name), flag)
+			logger.Debugf("overriding object store %q RGW config option %q (current value %q) with user-specified rgwConfig %q",
 				fmt.Sprintf("%s/%s", c.store.Namespace, c.store.Name), flag, currVal, val)
 		}
 		configOptions[flag] = val
 	}
 
+	for flag, val := range c.store.Spec.Gateway.RgwConfigFromSecret {
+		if val.Key == "" || val.Name == "" {
+			logger.Warningf("invalid object store %q rgwConfigFromSecret value %q=%+v", fmt.Sprintf("%s/%s", c.store.Namespace, c.store.Name), flag, val)
+			continue
```

Comment:
It's fine -- and probably best -- to return an error here. This is most a case where the user intended to add something but made a typo. IMO, it's helpful for the user if we throw a hard error that they have to handle. Otherwise, they won't know what's wrong unless they read opertor logs.

Reply:
changed to `return err`